### PR TITLE
Add command-line option to filter on available vaccination capacity

### PR DESCRIPTION
**Pull request by shakti-garg**
This PR add command line option "minCapacity" and related logic to filter appointments which have minimum available vaccination capacity. The default value is 1.

Real-world use case is that i am looking for 2 vaccination slots but getting lot of alerts with 1 slots.

### CODE REVIEW

File: search.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ type Appointments struct {
 		Sessions []struct {
 			SessionID         string   `json:"session_id"`
 			Date              string   `json:"date"`
-			AvailableCapacity float64  `json:"available_capacity"`
+			AvailableCapacity int      `json:"available_capacity"`
```
Comment (PrasadG193) [inline search.go]: @shakti-garg in some responses, we get float values. We switched to float to fix this. See - https://github.com/PrasadG193/covaccine-notifier/issues/1

Comment (shakti-garg) [reply search.go]: @PrasadG193 Got it. I have reverted the type back to float.

File: main.go [hunk 2]
```diff
@@ -55,6 +57,7 @@ func init() {
 	rootCmd.PersistentFlags().IntVarP(&interval, "interval", "i", getIntEnv(searchIntervalEnv), fmt.Sprintf("Interval to repeat the search. Default: (%v) second", defaultSearchInterval))
 	rootCmd.PersistentFlags().StringVarP(&vaccine, "vaccine", "v", os.Getenv(vaccineEnv), fmt.Sprintf("Vaccine preferences - covishield (or) covaxin. Default: No preference"))
 	rootCmd.PersistentFlags().StringVarP(&fee, "fee", "f", os.Getenv(feeEnv), fmt.Sprintf("Fee preferences - free (or) paid. Default: No preference"))
+    rootCmd.PersistentFlags().IntVarP(&minCapacity, "minCapacity", "m", getIntEnv(minCapacityEnv), fmt.Sprintf("Filter by minimum vaccination capacity. Default: (%v)", defaultMinCapacity))
```
Comment (PrasadG193) [inline main.go]:
```suggestion
    rootCmd.PersistentFlags().IntVarP(&minCapacity, "min-capacity", "m", getIntEnv(minCapacityEnv), fmt.Sprintf("Filter by minimum vaccination capacity. Default: (%v)", defaultMinCapacity))
```
Looks like there is indentation issue as well.

**Review by PrasadG193 — APPROVED** [review]
Looks good. I have a minor suggestion on flag name. We can merge it once addressed.

File: README.md [hunk 3]
```diff
@@ -33,16 +33,17 @@ Usage:
...
+  -a, --age int             Search appointment for age
+  -s, --state string        Search by state name
+  -d, --district string     Search by district name
+  -c, --pincode string      Search by pin code
+  -e, --email string        Email address to send notifications
+  -p, --password string     Email ID password for auth
+  -i, --interval int        Interval to repeat the search. Default: (60) second
+  -f, --fee string          Fee preferences - free (or) paid. Default: No preference
+  -v, --vaccine string      Vaccine preferences - covishield (or) covaxin. Default: No preference
+  -m, --minCapacity int     Filter by minimum vaccination capacity. Default: 1
```
Comment (PrasadG193) [inline README.md]:
```suggestion
  -m, --min-capacity int     Filter by minimum vaccination capacity. Default: 1
```

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**